angr/state_plugins/sim_action.py

```diff
@@ -233,6 +249,10 @@ def downsize(self):
     def all_objects(self):
         return [ a for a in [ self.addr, self.size, self.data, self.condition, self.fallback, self.fd ] if a is not None ]
 
+    @property
+    def is_symbolic(self):
+        return any([ getattr(a, "symbolic", False) for a in [ self.addr, self.size, self.data ] if a is not None ])
```

Comment:
This almost looks like black magic to me. Let's be more explicit:
```
return self.addr.symbolic or self.size.symbolic or self.data.symbolic
```

Reply:
Sure we can replace the `any` and be more explicit. I just followed the same style as `all_objects` and felt it looks better than having multiple `getattr` invocations. I think we should see how to change this based on how we deal with replacing `getattr`